torch/csrc/jit/runtime/static/ops.cpp

```diff
@@ -379,6 +379,10 @@ struct TEWrapper {
     cg->call(args);
   }
```

Comment:
Can we rip out the `operator()` overload if we're not going to use it anymore?

Reply:
We probably can

Reply:
Done!